### PR TITLE
Fix translate_validation: use params

### PR DESCRIPTION
**Pull request by blueyed**
Related: https://github.com/encode/django-rest-framework/pull/5785

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
     verbose_field_name,
     verbose_lookup_expr
 )
+from django_filters.rest_framework import MultipleChoiceFilter
```
Comment (blueyed) [inline tests/test_utils.py]: Have a fixup for this already.

**Comment by blueyed** [comment]
Gentle ping.

**Comment by blueyed** [comment]
@rpkilby 
Thanks for the review.

@carltongibson 
Can we merge and release this, please?
This is holding me back from upgrading django-filter and therefore Django itself.

I will eventually have to install django-filter from a fork just because of that, but would like to avoid that of course.

**Comment by carltongibson** [comment]
Hi @blueyed. 

Pip’s VCS support is first class. You should go ahead with that if it’s holding you back. 

I’m looking at rolling 2.1 over the holiday period.

**Comment by carltongibson** [comment]
Oh, and I’ve seen this PR about 400 times but the lack of a description meant I’ve not actually reviewed it.

**Comment by blueyed** [comment]
> Pip’s VCS support is first class

Well, not so much with hashes etc. anyway, it would work for me, but I would just like to avoid it and this is a decent bugfix after all.

**Comment by carltongibson** [comment]
> ...this is a decent bugfix after all.

That I'm sure it is. I'm just yet to decipher it. 😃

As I say, 2.1 for the holiday period.

**Comment by blueyed** [comment]
\o/
Thanks!

**Pull request merged**